tests/sdk/test_cancel.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Owkin, inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import datastore
+from .utils import mock_requests
+
+
+def test_cancel_compute_plan(client, mocker):
+    item = datastore.COMPUTE_PLAN
+    method = client.cancel_compute_plan
```

Comment:
Very minor: I think there is no need to create variable for these 2 objects.

Reply:
Indeed, that's cleaner this way 👍